src/testRunner/unittests/tscWatch/emitAndErrorUpdates.ts

```diff
@@ -279,10 +279,10 @@ export class Data {
         };
         const lib1ToolsPublic: File = {
             path: `/user/username/projects/myproject/lib1/tools/public.ts`,
-            content: `export * from "./tools.interface";`
+            content: `export * from "./toolsinterface";`
         };
         const lib1ToolsInterface: File = {
-            path: `/user/username/projects/myproject/lib1/tools/tools.interface.ts`,
+            path: `/user/username/projects/myproject/lib1/tools/toolsinterface.ts`,

```

Comment:
Are filenames like this problematic now?

Reply:
No, they just add a (higher priority) lookup location which has knock-on effects regarding file watchers in the baseline, so to avoid baseline noise Sheetal asked I simply change the filename.